projects/solar/inc/fan_control_solar.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <stdbool.h>
```

Comment:
By convention, all our header files start with a comment like this:
```c
// A short description of the module.
// Requires X, Y, and Z to be initialized.

// An optional extended description of the module.
```
This helps knowing what each module does, and the `// Requires ... to be initialized` comments should list all the modules/libraries it depends on (including transitively - if A requires B and B requires C then A should list C in the requirement list) to help with initializing everything that's required for tests.

(Also, there should be a blank line after the <stdbool.h> include according to the coding standards.)